github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -23,6 +23,18 @@
   when: item.status == true
   with_items: env.post_settings
 
+- name: Checking a domain_access is enabled
+  shell: 'drush pml --status=enabled -l {{ site_url }} | grep -c "(domain)" || true'
```

Comment:
@Sanchiz Hi, I will be in touch in my skype if you will have a little bit free time or any idea.
Have a nice day )

/cc @podarok